src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -186,26 +184,39 @@ void createConsumerWithForwardedHeaderWrongProto(VertxTestContext context) throw
         assertTrue(context.awaitCompletion(TEST_TIMEOUT, TimeUnit.SECONDS));
     }
 
-    @ParameterizedTest
-    @CsvFileSource(resources = "/ConsumerWithWrongParameter.csv")
-    void createConsumerWithWrongParameter(String key, String value, String message, int httpCode, VertxTestContext context) throws InterruptedException {
-        JsonObject json = new JsonObject();
-        json.put("name", name);
-        json.put(key, value);
+    @Test
+    void createConsumerWithWrongAutoOffsetReset(VertxTestContext context) throws InterruptedException, TimeoutException, ExecutionException {
+        checkCreatingConsumer("auto.offset.reset", "foo", HttpResponseStatus.INTERNAL_SERVER_ERROR,
+                "Invalid value foo for configuration auto.offset.reset: String must be one of: latest, earliest, none", context);
 
-        consumerService().createConsumerRequest(groupId, json)
-            .sendJsonObject(json, ar -> {
-                context.verify(() -> {
-                    assertTrue(ar.succeeded());
-                    HttpResponse<JsonObject> response = ar.result();
-                    assertEquals(httpCode, response.statusCode());
-                    HttpBridgeError error = HttpBridgeError.fromJson(response.body());
-                    assertEquals(httpCode, error.getCode());
-                    assertEquals(message, error.getMessage());
-                });
-                context.completeNow();
-            });
+        context.completeNow();
+        assertTrue(context.awaitCompletion(TEST_TIMEOUT, TimeUnit.SECONDS));
+    }
+
+    @Test
+    void createConsumerWithWrongEnableAutoCommit(VertxTestContext context) throws InterruptedException, TimeoutException, ExecutionException {
+        checkCreatingConsumer("enable.auto.commit", "foo", HttpResponseStatus.INTERNAL_SERVER_ERROR,
+                "Invalid value foo for configuration enable.auto.commit: Expected value to be either true or false", context);
 
+        context.completeNow();
+        assertTrue(context.awaitCompletion(TEST_TIMEOUT, TimeUnit.SECONDS));
+    }
+
+    @Test
+    void createConsumerWithWrongFetchMinBytes(VertxTestContext context) throws InterruptedException, TimeoutException, ExecutionException {
+        checkCreatingConsumer("fetch.min.bytes", "foo", HttpResponseStatus.INTERNAL_SERVER_ERROR,
+                "Invalid value foo for configuration fetch.min.bytes: Not a number of type INT", context);
+
+        context.completeNow();
+        assertTrue(context.awaitCompletion(TEST_TIMEOUT, TimeUnit.SECONDS));
+    }
+
+    @Test
+    void createConsumerWithWrongRequestTimeoutMs(VertxTestContext context) throws InterruptedException, TimeoutException, ExecutionException {
+        checkCreatingConsumer("request.timeout.ms", "30000", HttpResponseStatus.BAD_REQUEST,
```

Comment:
well this test should pass, because 30000 ms is a valid value for a timeout in ms configuration parameter.
As you look at the error message it's about OpenAPI validation because the right parameter name is `consumer.request.timeout.ms` and not just `request.timeout.ms`. After fixing it, you should use a value which is not a valid timeout (i.e. just "foo") for doing the test.

Reply:
I'm happy that you highlighted this scenario! I don't want to check `consumer.request.timeout.ms` in this scenario as you mentioned before. I want to check validation error for wrong JSON `KEY` (not value). This is a reason why I have expected error message as `Validation error on: body - $.request.timeout.ms...`

Reply:
so the method name is misleading :-)
It should not be `createConsumerWithWrongRequestTimeoutMs` but more `createConsumerWithWrongConfigParameter` or `createConsumerWithNotExistingParameter`. The name it has now seems to relate to the value, so wrong value for the request timeout parameter.

Reply:
okay, if you don't mind I will call it like `createConsumerWithWrongKeyConsumerRequestTimeoutMs`. Let's say that in JSON data has two types, key and value and having `Key` at the method name should be more clear that we check validation for syntax `consumer.request.timeout.ms`

Reply:
Pushed changes

Reply:
Tbh checking that the name for `request.timeout.ms` is wrong is like checking that you are using a "wrong parameter name". Using the request timeout ms as a reference is misleading.

Reply:
Well, if we have an error message for this case? why do you think this test can be misleading? If we have some logic at the back-end we have to check it. thanks

Reply:
Testing that you are using a wrong name for request timeout is like testing that you are providing a wrong parameter name. It's a superset of the specific case you are testing. For this reason it just makes sense testing the usage of a wrong parameter name; you tell will just be a specific case of that.

Reply:
So, what do you recommend to do?

Reply:
changing the test method in `createConsumerWithNotExistingParameter` and using `foo` or something else as parameter name

Reply:
thanks, done